### PR TITLE
Adds termination to .bin

### PR DESCRIPTION
**Pull request by dpetrisko**
This PR adds a termination string to the CCE ucode so that loaders know where to stop

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,7 +66,9 @@ dirs:
 	./$(AS) -d -i $< -o $@
 
 %.bin: %.mem
-	xxd -r -p $< > $@
+	sed -i -e '$$a1111111111111111111111111111111111111111111111111111111111111111' $^
+	sed -i -e '$$a1111111111111111111111111111111111111111111111111111111111111111' $^
+	perl -ne 'print pack("B64", $$_)' < $^ > $@
```
Comment (muwyse) [inline Makefile:74]: A few questions:

1. What determines the width of this string?
2. Are the "$$" there for matching by the perl command?
3. Why is the rest of the string "a111....1"? What does the "a" mean?
4. Perhaps add a comment above the Make target to explain the termination and why it is needed. It is not obvious from the code in the target.

Comment (dpetrisko) [reply Makefile:74]: 1. 64, it's the length of a padded word in the ucode
2/3. $a is sed syntax for appending to the end of a file. The $$ is used to escape the $ in the makefile bashstring
4. Added!

**Review by muwyse — APPROVED** [review]
LGTM!

Bumping bedrock in the SDK for this PR will depend on the top_dev changes, specifically to the NBF loader/bootrom, correct?

**Comment by dpetrisko** [comment]
Nope this should be harmless without top_dev

**Pull request merged**